src/lib/components/ServerAddressModal.svelte

```diff
@@ -68,35 +95,44 @@
           for="address"
           class="block mb-2 text-sm font-medium text-gray-300 text-left"
           >Socket.IO Server Address *</label
-        >    
+        >
+
         <input
           type="text"
           name="address"
+          id="address"
           pattern="(http|https|ws|wss):\/\/(.)*"
           bind:value={serverAddress}
-          oninvalid={()=>"this.setCustomValidity('URL should start with one of the http|https|ws|wss:// ')"}
+          oninvalid={() =>
+            "this.setCustomValidity('URL should start with one of the http|https|ws|wss:// ')"}
           placeholder="example: http://localhost:3000"
           class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-600 dark:border-gray-500 dark:placeholder-gray-400 dark:text-white"
           required
         />
-      </div> 
+      </div>
       <div>
+        <p id="tabuse" class="sr-only">
+          , If you press ctrl +m you can switch between inserting spaces or the
```

Comment:
I guess "," is redundant. By the way, we'd better add an instruction to the help.

Reply:
okay wait, so class sr-only is not working? because this content is only for screen readers, and it should not be shown for you when you go to the site. The paragraph is needed because it has an id and it is verbalized when acceeding to the textarea. It's like the ctrl m thing, it should not disturb you because the normal behaviour is the one  you put, but if a screenreader user arrives to that input he can know that there exists an alternative to use tab. So the checkbox is a great idea, but just curious. the times where I added sr-only, the text is appearing for you when you execute the project?? I thought tailwind had a class sr-only already, but if not I can import it manually so most of the problems get solved.